frontend/src/redux/actions.js

```diff
@@ -54,9 +54,9 @@ export const setFinancesValue = (valueName, value) => ({
   payload: { valueName, value },
 });
 
-export const setShoppingListValue = (index, valueName, value) => ({
-  type: "setShoppingListValue",
-  payload: { index, valueName, value },
+export const setShoppingValue = (value) => ({
```

Comment:
i would call this 'setShopping' not 'setShoppingValue' because its redundant (the other action names with 'value' were for updating general values in the state)